src/main.rs

```diff
@@ -262,7 +261,7 @@ fn main() {
                     if sigquit.load(Ordering::Relaxed) {
                         panic!("SIGQUIT received");
                     }
-                    if e.errno() == Errno::ESTRPIPE {
+                    if e.errno() == libc::ESTRPIPE {
```

Comment:
Please double check that this is correct, I don't 100% grok what happened in https://github.com/diwic/alsa-rs/commit/fbe82eff4bfd5592b80ac5f40a1ce5f0dc78f3a3

Reply:
LGTM on first glance. They got rid of `nix` which they used to reexport `nix::Error` as `Errno`, so now we just get the constants from `libc`. I will confirm in a bit.